### ISSUE TITLE
Improvements in 9.3. Add-ons Manager documentation

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4540]
**Reported by blindbhavya on 2014-10-10 16:53**
Following are suggestions for the 9.3. Add-ons Manager section of the User Guide
1 The Add-ons Manager, accessed by selecting Manage add-ons under Tools in the NVDA menu, allows you
A Perhaps, you could add the string of text ‘which can be’ and remove the comma before ‘accessed’ and perhaps replace ‘accessed’ with ‘launched’. (completely optional)
B You may want to add ‘...’ after ‘Manage Addons’
C You may want to add ‘the’ before ‘Tools’ and ‘submenu’ after ‘Tools’
2 even provide support for extra Braille displays or speech synthesizers.
AIf you think ‘even’ is not required you may remove it. (optional)
bSomething like ‘add or enhance support for applications that are otherwise inaccessible’ should be added.
C Perhaps, ‘extra’ may be replaced by ‘third party’ or ‘additional’ (completely optional)
3 A package name, version and author are shown
A ‘A’ should be replaced with ‘The’
B It should be stated that the author’s e-mail address is also present.
C If I am not wrong, then to be grammatically correct, ‘are’ should be replaced with ‘is’ 
4 Somewhere in the documentation it should be stated that to install add-ons successfully, NVDA must be running.
5 It should be documented that if a version of the add-on the user is attempting to install is already installed, NVDA will ask the user whether they want to update that version with the add-on currently being installed.
Hope I was clear and detailed enough
P.S. I filed another documentation related ticket, since I have quite a few suggestions for this section (out of which some definitely are completely optional).

**Comment by bhavyashah** [comment]
Feel free to close this ticket...
